docs/5.0/admin-guide.md

```diff
@@ -1268,15 +1268,52 @@ $ tctl status
 ## Ansible Integration
 
 Ansible uses the OpenSSH client by default. This makes it compatible with
-Teleport without any extra work, except configuring OpenSSH client to work with
-Teleport Proxy:
+Teleport without any extra work, except configuring OpenSSH client to work with Teleport Proxy.
 
-* configure your OpenSSH to connect to Teleport proxy and use `ssh-agent` socket
-* enable scp mode in the Ansible config file (default is `/etc/ansible/ansible.cfg` ):
 
-```bash
-scp_if_ssh = True
-```
+There are two recommended scenarios:
+
+1. Ansible connects directly to Teleport Node using built in ssh library over port 3022 and Teleport user credentials
+
+2. Ansible connects to Teleport Node using the Teleport Proxy as a jumphost. This method uses Teleport user credentials over port 3022
+
+The configuration for openssh is as follows:
+
+1. Add hosts to known hosts in Ansible node
+    * From Teleport Auth Node:
+        ```bash
+        $ tsh auth export --type=host
+        ```
+    * Add the output on Ansible node `/.ssh/known_hosts` by modifying the node section to match the nodes you are reaching (note: wildcards are accepted).
+    ```bash
+    @cert-authority <NODE(S)> ssh-rsa AAA...REDACTED...TD9 logins=-teleport-nologin-f0d...REDACTED...fb1&type=host
+    ```
+    * Modify ssh config file to include the proxy node and the desired Teleport nodes with `ProxyCommand` or `ProxyJump` parameters configured
+
+1. Create identity file and extract to Ansible node
```

Comment:
Given the above comment about restructuring the layout, I think this section will need to be updated to reflect the options. 

Examples:

Connection Type (include very simple and minimal ~/.ssh/config examples)
- Direct to Node
- Proxy to Node

Authentication Type
- `tsh` credentials
- `tctl auth sign` long lived SSH creds